modules/real_time_enforcer/main.tf

```
@@ -113,35 +115,65 @@ resource "google_storage_bucket_object" "enforcer_policy" {
 #-----------------------#
 
 data "template_file" "cloud-init" {
-  template = "${file("${path.module}/templates/cloud-init.yml")}"
+  template = file("${path.module}/templates/cloud-init.yml")
 
-  vars {
-    project_id        = "${var.project_id}"
-    enforcer_bucket   = "${google_storage_bucket.main.name}"
-    subscription_name = "${google_pubsub_subscription.main.name}"
+  vars = {
+    project_id        = var.project_id
+    enforcer_bucket   = google_storage_bucket.main.name
+    subscription_name = google_pubsub_subscription.main.name
   }
 }
 
 resource "google_compute_instance" "main" {
-  name = "${local.enforcer_name}"
-  zone = "${local.enforcer_zone}"
+  name = local.enforcer_name
+  zone = local.enforcer_zone
 
-  project                   = "${var.project_id}"
-  machine_type              = "${var.enforcer_type}"
+  project                   = var.project_id
+  machine_type              = var.enforcer_type
   allow_stopping_for_update = true
 
   boot_disk {
     initialize_params {
-      image = "${var.enforcer_boot_image}"
+      image = var.enforcer_boot_image
+    }
+  }
+  dynamic "network_interface" {
+    for_each = [local.network_interface]
```

`local.network_interface` is already a list.

```suggestion
    for_each = local.network_interface
```